### PR TITLE
Feature fix chef sign up

### PR DESCRIPTION
**Pull request by megangilpin**

### CODE REVIEW

File: server/models/conversation.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const conversationSchema = new Schema({
+    // Which is better?
+    // option A
+    userId1: { type: Schema.Types.ObjectId, ref: "User" },
+    userId2: { type: Schema.Types.ObjectId, ref: "User" },
+    // option B
+    participants: [{ type: Schema.Types.ObjectId, ref: "User" }],
```
Comment (sagar-13) [inline server/models/conversation.js:10]: I vote option B.

**Review by sagar-13 — APPROVED** [review]
Looks good to me

File: client/src/components/MealCard.js [hunk 2]
```diff
@@ -77,24 +85,41 @@ function MealCard({
...
+                        component="img"
+                        alt="meal1"
+                        height="150"
+                        image={picURL}
+                        title="meal1"
+                    />
+                ) : (
+                    <React.Fragment>
+                        <CardContent className={classes.replacement}>
+                            {/* <Box className={classes.replacement}> */}
```
Comment (ThinkTink) [inline client/src/components/MealCard.js]:
```suggestion
```

File: client/src/components/MealCard.js [hunk 3]
```diff
@@ -77,24 +85,41 @@ function MealCard({
...
+                            <Typography
+                                color="primary"
+                                align="center"
+                                variant="subtitle2"
+                            >
+                                The chef is still cooking up an image
+                            </Typography>
+                            {/* </Box> */}
```
Comment (ThinkTink) [inline client/src/components/MealCard.js]:
```suggestion
```

File: server/routes/users.js [hunk 4]
```diff
@@ -53,20 +53,25 @@ router.put(
...
 
-        const user = await usersController.sanatize(
-            await usersController.update(id, req.body)
-        );
+        let chefProfile = {};
 
-        if (req.body.isChef && !user.isChef) {
+        if (req.body.newChef && !req.body.isChef) {
+            const user = await usersController.findOneWithId(id);
+            console.log(user);
```
Comment (ThinkTink) [inline server/routes/users.js]:
```suggestion
```

File: server/routes/users.js [hunk 5]
```diff
@@ -53,20 +53,25 @@ router.put(
             return;
         }
 
-        const user = await usersController.sanatize(
-            await usersController.update(id, req.body)
-        );
+        let chefProfile = {};
```
Comment (ThinkTink) [inline server/routes/users.js]: This can be const and declared at the time its assigned below

File: server/routes/users.js [hunk 6]
```diff
@@ -53,20 +53,25 @@ router.put(
...
                 cuisineSpecialty: [...req.body.cuisineSpecialty],
-                userId: id,
+                userId: user._id,
             };
-            const chefData = await chefsController.create(data);
-            delete chefData.userId;
-            user.chefProfile = chefData;
+            chefProfile = await chefsController.create(data);
+            delete chefProfile.userId;
+            req.body.isChef = !req.body.isChef;
```
Comment (ThinkTink) [inline server/routes/users.js]: Unless this is actually something you can toggle back and forth, I'd rather it be more explicit (`.isChef = true`)

File: server/models/message.js [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+const Schema = mongoose.Schema;
+
+const messageSchema = new Schema({
+    sender: { type: Schema.Types.ObjectId, ref: "User" },
+    receiver: { type: Schema.Types.ObjectId, ref: "User" },
+    message: { type: String },
+    createdAt: { type: Date, default: Date.now },
+});
+
+module.exports = messageSchema;
```
Comment (ThinkTink) [inline server/models/message.js:11]: We can close that other PR if these files will be merged with this one

**Review by ThinkTink — APPROVED** [review]

**Pull request merged**